src/client/css/nav.css

```diff
@@ -2,40 +2,63 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-nav {
+nav.site-nav {
   position: fixed;
   top: var(--header-h);
   left: 0;
   height: calc(100vh - var(--header-h));
   display: inline-flex;
   flex-flow: column nowrap;
-  padding: var(--padding-lg) 0 var(--footer-h);
+  padding: var(--padding-lg) var(--padding-lg) var(--footer-h);
   background: white;
   border-right: 1px solid var(--gray-10);
   border-image: linear-gradient(#0000 3%, var(--gray-10) 5%, var(--gray-10) 95%, #0000 97%) 1;
   z-index: 1;
 }
 
-nav[hidden] {
+nav.site-nav[hidden] {
   display: none;
 }
 
-nav > a {
-  padding: var(--padding-sm) var(--padding-lg);
-  text-decoration: none;
+nav.site-nav .pages-nav,
+nav.site-nav .meta-nav {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  justify-content: space-evenly;
+  gap: var(--padding-sm);
```

Comment:
Question: What do you think of not using `gap` to space the link items and instead giving `nav-item` a higher vertical padding? That would increase the clickable area and visually align the hover highlight with the height of the “active indicator” (the stripe purple stripe on the left side).

Reply:
Good idea, fixed in 513347c5.